working/templates/package-project/GettingStarted.md

```diff
@@ -0,0 +1,154 @@
+# Getting Started with Skyline DataMiner DevOps
+
+Welcome to the Skyline DataMiner DevOps environment!  
+This quick-start guide will help you get up and running.  
+For more details and comprehensive instructions, please visit [DataMiner Docs](https://docs.dataminer.services/).
+
+<!--#if (CreateDataMinerPackage)-->
+## Creating a DataMiner Application Package
+
+This project is configured to create a `.dmapp` file every time you build the project.  
+Simply compile or build the project, and you will find the generated `.dmapp` in the standard output folder, typically the `bin` folder of your project.
+
+When publishing, this project will become its own item on the online catalog.
+
+## The DataMiner Package Project
+
+This project is designed to create multi-artifact packages in a straightforward manner.
+
+### Other DataMiner Projects in the Same Solution
+
+Every **Skyline.DataMiner.SDK** project, except other DataMiner Package Projects, will by default be included within the `.dmapp` created by this project.  
+You can customize this behavior using the **PackageContent/ProjectReferences.xml** file. This allows you to add filters to include or exclude projects as needed.
+
+### Adding Content from the Catalog
+
+You can reference and include additional content from the catalog using the **PackageContent/CatalogReferences.xml** file provided in this project.
+
+### Importing from DataMiner
+
+You can import specific items directly from a DataMiner agent:  
+
+1. Connect to an agent via **Extensions > DIS > DMA > Connect**.
+2. Once connected, you can import specific DataMiner artifacts.
```

Comment:
This doesn't seem to be an actual step the user should carry out, but rather additional info about step 1. I'd therefore recommend to make this an unnumbered, indented sentence below step one, and to change the step below this to step 2.